remoteit/desktop
# [DESK-590][DESK-610] Launch smb type and fix missing data bug

### Description


Update graphql api url
Styling fixes
Device list cleanup and handling of duplicate 'this' device
Adding Samba launch type
Fix crashing when no last reported date
When signing in always default to device list

### Checklist




### Test plan



1. Sign in
2. Go to a device with smb
3. Try to launch it
4. You should see it launch with smb://

### Ticket



<https://remoteit.atlassian.net/browse/DESK-610>
<https://remoteit.atlassian.net/browse/DESK-590>
